### DISCUSSION TITLE
Custom placeholders for Towns

### DISCUSSION THREAD

**Discussion by FARF12** [TownyAdvanced/Towny#5593] [Q&A]
Hello LlmDl

I would like to ask about the possibility of creating several custom placeholders for the town.

I think having this feature would improve the quality of my server and other server owners who would be able to use it.After all, the presence of several custom placeholders can be used to develop new game mechanics inside Towny

**Comment by LlmDl** [comment]
Well don't keep this idea a big secret any longer, go on...

**Reply by FARF12** [reply]
I wanted to make a system of dynamic "ideologies".There would be 3 aspects "Militarization, Economy, Statehood".The value of these aspects cannot exceed 5.

For example
If militarization is equal to 1, then access opens to the ability to produce guns, etc
. If the economy is equal to one, then there are more slots at auction and for citizens, etc.
It's all easy for me to do, but the problem is that the values of these aspects need to be stored somewhere.After all, each town has its own aspects.Here I also had a question where?If there are literally several placeholders, the value of which will be equal to 0, then this can be implemented.

**Reply by LlmDl** [reply]
Towny's metadata API might be something you want to learn about. Sounds like a fair bit of work.
